RecoBTag/TensorFlow/test/test_deepvertex_cfg.py

```diff
@@ -0,0 +1,77 @@
+
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.PatAlgos.tools.helpers import getPatAlgosToolsTask
+
+process = cms.Process("PATtest")
+
+## MessageLogger
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+## Options and Output Report
+process.options = cms.untracked.PSet( wantSummary = cms.untracked.bool(True) )
+
+## Source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring()
+)
+## Maximal Number of Events
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(100) )
+
+## Geometry and Detector Conditions (needed for a few patTuple production steps)
+process.load("Configuration.Geometry.GeometryRecoDB_cff")
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc')
+process.load("Configuration.StandardSequences.MagneticField_cff")
+
+## Output Module Configuration (expects a path 'p')
+from PhysicsTools.PatAlgos.patEventContent_cff import patEventContentNoCleaning
+process.out = cms.OutputModule("PoolOutputModule",
+                               fileName = cms.untracked.string('patTuple.root'),
+                               ## save only events passing the full path
+                               #SelectEvents = cms.untracked.PSet( SelectEvents = cms.vstring('p') ),
+                               ## save PAT output; you need a '*' to unpack the list of commands
+                               ## 'patEventContent'
+                               outputCommands = cms.untracked.vstring('drop *', *patEventContentNoCleaning )
+                               )
+
+patAlgosToolsTask = getPatAlgosToolsTask(process)
+process.outpath = cms.EndPath(process.out, patAlgosToolsTask)
+
+## and add them to the event content
+from PhysicsTools.PatAlgos.tools.jetTools import updateJetCollection
+
+updateJetCollection(
```

Comment:
I'm missing modifications to the ```pfDeepFlavourTagInfos*``` in parameters ```run_deepVertex``` and ```compute_probabilities```. Isn't it required for a proper test?